### PR TITLE
Change ShoppingList From Modal to Page

### PR DESCRIPTION
**Pull request by MarilynY**

### CODE REVIEW

File: components/ShoppingListPage.js [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+            checkedBox: [],
+            items: [],
+            purchasedIngredient: [],
+            userId: this.props.userId,
+        }
+    }
+
+    onPressYes = () => {
+        Actions.home({page: "inventory"});
+    }
```
Comment (OswaldHe) [inline components/ShoppingListPage.js]: If the user does not check anything, it is better to redirect back to meal plan page and add a toast as a notification that no ingredient was added to inventory

File: components/ShoppingListPage.js [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+                        method: "POST",
+                        headers: {
+                            'Accept': 'application/json',
+                            'Content-Type': 'application/json',
+                            'Authorization':  accessToken
+                        },
+                        body: JSON.stringify(this.state.purchasedIngredient)
+                    }).then((response)=>{
+                        if(response.status == "200"){
+                            Alert.alert(
```
Comment (OswaldHe) [inline components/ShoppingListPage.js]: Use toast instead of alert

File: components/ShoppingListPage.js [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+                if (this.state.items[i].id === id) {
+                    volume = this.state.items[i].volume;
+                    break;
+                }
+            }
+
+            this.state.purchasedIngredient.push({
+                ingredientId : id,
+                inventoryVolume: volume,
+                unitsOfMeasure: 13,
```
Comment (OswaldHe) [inline components/ShoppingListPage.js]: It should not be 13 since 13 represent pint, not gram.

File: components/ShoppingListPage.js [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+                                        alignSelf:'center',
+                                        justifyContent:'center',
+                                        backgroundColor:"deepskyblue",
+                                        width: 360,
+                                        marginTop: 20
+                                    }}
+                                    onPress={()=>{
+                                        this.onFinishShoppingButton()
+                                    }}>
+                                    <Text >Finish Shopping</Text>
```
Comment (OswaldHe) [inline components/ShoppingListPage.js]: Change the style of text to white and bold

File: components/ShoppingListPage.js (hunk 1 above)
Comment (MarilynY) [reply components/ShoppingListPage.js]: OK, good idea.

File: components/ShoppingListPage.js (hunk 3 above)
Comment (MarilynY) [reply components/ShoppingListPage.js]: I forgot which one is gram. Please help me change it.

File: components/ShoppingListPage.js (hunk 4 above)
Comment (MarilynY) [reply components/ShoppingListPage.js]: solved

**Pull request merged**